Dockerfile

```diff
@@ -4,6 +4,11 @@ COPY . /wal2json
 WORKDIR /wal2json
 RUN make && make install
 
-FROM postgres:9.6-alpine
+FROM postgres:9.6-alpine as db
 COPY --from=build /usr/local/lib/postgresql/wal2json.so /usr/local/lib/postgresql/
 COPY /docker-entrypoint-initdb.d /docker-entrypoint-initdb.d
+
+FROM db as testing
```

Comment:
Not that if anyone builds this image without the using a build target they will end up with this installed as well. Maybe create a separate Dockerfile for testing?

Reply:
actually this image is only supposed to be used only for testing. We are finally not going to package wal2json together with a postgres image. 
So probably I should remove the label and only have testing.